### PR TITLE
Migrate ccs v3 to @adobe/css-tools v4

### PR DESCRIPTION
**Pull request by holblin**
**What**:

Migrate the dependency `css` to `@adobe/css` to reduce maintained dependencies.

**Why**:

CSS package is not maintained anymore and has some sub-dependencies which are deprecated.

**How**:

Change the import

**Checklist**:

- [X] Documentation N/A
- [X] Tests N/A
- [X] Updated Type Definitions N/A
- [X] Ready to be merged

### CODE REVIEW

**Comment by holblin** [comment]
Note: The version 4 of `@adobe/css-tools` is very recent. I used `import {parse} from '@adobe/css-tools/dist/cjs/cssTools'` to make sure it load the `cjs` build and not the `umd` build. If someone knows why `jest-dom` try to load the `umd` build instead of the `cjs` and as a solution, feel free to share :-)

**Review by MichaelDeBoey — APPROVED** [review]

**Review by kentcdodds — APPROVED** [review]
LGTM, but I'm not a current maintainer. If nobody merges in the next week or so though I'll go ahead and merge

File: src/utils.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import redent from 'redent'
-import cssParse from 'css/lib/parse'
```
Comment (xfix) [inline src/utils.js:2]: Directly loading `dist/cjs/cssTools` feels somewhat hacky as there is no guarantee this path will exist in future releases of `@adobe/css-tools`. I wonder if using `require` instead would work.

Comment (holblin) [reply src/utils.js:2]: I agree but I can't found a way to load the CJS build.

If I do without a path:
```
    ReferenceError: self is not defined

      1 | import redent from 'redent'
    > 2 | import {parse} from '@adobe/css-tools'
        | ^
```
If I do a require:
```
   ReferenceError: self is not defined

      1 | import redent from 'redent'
      2 | import isEqual from 'lodash/isEqual'
    > 3 | const {parse} = require('@adobe/css-tools')
        |                 ^
```

Let me know if you have a better solution or something that I could change for the build of `@adobe/css-tools`

Comment (holblin) [reply src/utils.js:2]: I made a change of how `@adobe/css-parse` is build (UMD build, globalThis is now set to this in wepack config). This should solve the issue/concern.

**Comment by holblin** [comment]
@kentcdodds I updated the import to solve the CJS import path that I was using :-)
Could you merge it when you get some time? 😇 Thanks!

**Review by kentcdodds — APPROVED** [review]
Thank you 👍

**Pull request merged**